libclc/generic/lib/math/atan.cl

```diff
@@ -181,3 +181,15 @@ _CLC_OVERLOAD _CLC_DEF double atan(double x)
 _CLC_UNARY_VECTORIZE(_CLC_OVERLOAD _CLC_DEF, double, atan, double);
 
 #endif // cl_khr_fp64
+
+#ifdef cl_khr_fp16
```

Comment:
Can this be `_CLC_DEFINE_UNARY_BUILTIN_FP16(atan)`?

Reply:
I agree, but I had some weird compilation errors. Does it work on your side?

Reply:
Alright, just found that it is due to includes order in `atan.cl`. I'll change that